rmgpy/data/solvation.py

```diff
@@ -1795,23 +1820,22 @@ def get_solvation_correction(self, solute_data, solvent_data):
         correction.entropy = self.calc_s(correction.gibbs, correction.enthalpy)
         return correction
 
-    def get_Kfactor(self, solute_data, solvent_data, T):
+    def get_Kfactor(self, delG298, delH298, delS298, solvent_name, T):
         """
-        Given solute_data, solvent_data, and temperature, calculates K-factor T
-        if the solvent's name_in_coolprop is not None. K-factor = y_solute / x_solute.
+        Given delG298 (J/mol), delH298 (J/mol), delS298 (J/mol/K), solvent_name, and temperature T,
+        calculates K-factor at T.  K-factor = y_solute / x_solute.
         If the temperature is above the critical temperature of the solvent, it raises InpurError.
-        If the solvent's name_in_coolprop is None, it raises DatabaseError
+        If the given solvent_name is not available in CoolProp, it raises DatabaseError.
         """
```

Comment:
Add an args block here. You can also move the error information to a raises block if you want